docker/Dockerfile.build

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /usr/src
+
+RUN set -ex \
```

Comment:
I'm not used to this style of including `set -ex` in front of the various `RUN` commands in a Dockerfile. Docker will fail a build if a commands exits with a non-zero code, so I don't see why the `set -e` is needed. I guess the `set -x` bit might add some clarity when you have multiple commands chained together in one `RUN`. I've just not seen that style used. However, I have no functional complaints about that bit.